src/pages/Login/LoginPage.tsx

```diff
@@ -155,13 +187,15 @@ export class LoginPage extends React.Component<LoginProps, LoginState> {
 
     const messages = this.getHelperMessage();
     const isLoggingIn = this.props.isPostLoginPerforming || this.props.status === LoginStatus.logging;
+    const isLoginBtnDisabled =
+      isLoggingIn || (this.props.postLoginErrorMsg !== undefined && this.props.postLoginErrorMsg.length !== 0);
 
     // Unfortunately, typescripg typings are wrong in the PatternFly
     // library. So, this casts LoginForm as "any" so that it is
     // possible to use the "isLoginButtonDisabled" property.
-    const Form = LoginForm as any;
+    const CredentialsForm = LoginForm as any;
```

Comment:
This could be a little bit more typesafe.

Reply:
I see the typing issue on LoginForm is no longer present. So, I removed that cast, and also the preceding comment.